Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -481,12 +481,8 @@
     solutions:
       food:
         maxVol: 20
-  - type: SolutionSpiker
-    sourceSolution: food
-  - type: Extractable
-    grindableSolutionName: food
-  - type: StaticPrice
-    price: 0
+  - type: ExplosionResistance
+    damageCoefficient: 0.025
```

Comment:
you should probably add a comment here about exactly how strong this is supposed to be

Reply:
change "and not minibombs and nukes" to "but not minibombs and nukes"